src/unix/core.c

```diff
@@ -52,6 +53,9 @@
 # include <mach-o/dyld.h> /* _NSGetExecutablePath */
 # include <sys/filio.h>
 # include <sys/ioctl.h>
+# ifndef LOGIN_NAME_MAX
+#  define LOGIN_NAME_MAX _POSIX_LOGIN_NAME_MAX
```

Comment:
This looks incorrect to me.  Most platforms leave _POSIX_LOGIN_NAME_MAX at the POSIX default of 9 (8 characters + zero byte) even though their real maximum is much larger.